sct-commons/src/main/java/org/lfenergy/compas/sct/commons/scl/ied/AbstractLNAdapter.java

```diff
@@ -493,11 +492,10 @@ public List<ResumedDataTemplate> getDAI(ResumedDataTemplate rDtt, boolean updata
         List<ResumedDataTemplate> resumedDTTs = lNodeTypeAdapter.getResumedDTTs(rDtt);
 
         // complete the list by the overridden information in DAI
```

Comment:
useless comment

Reply:
OK, I removed the comment but, I renamed the method `completeResumedDTTFromDAI` to `overrideAttributesFromDAI` to avoid losing information.